lib/dispatcher.rb

```diff
@@ -10,11 +10,18 @@ def email_observers(cart)
     end
   end
 
+  def email_already_approved_with_changes(cart)
+    cart.approved_approvals.each do |approval|
+      CommunicartMailer.cart_notification_email(approval.user_email_address, approval, false).deliver
+    end
+  end
+
   def email_sent_confirmation(cart)
     CommunicartMailer.proposal_created_confirmation(cart).deliver
   end
 
   def deliver_new_cart_emails(cart)
```

Comment:
Question: This method doesn't appear to be sending new emails to approvers. With this newest change, it will send to 1) approvers that have already approved, 2) observers, and 3) the requester.

Am I missing something? Perhaps the sending of emails to the rest of approvers is happening elsewhere.